config/700-config-logging.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
Can you move file name from `config/700-config-logging.yaml` to `config/301-config-logging.yaml`

Otherwise PAC controller is crashing with below error
```
{"severity":"EMERGENCY","timestamp":"2023-04-26T06:15:11.256553352Z","logger":"pipelinesascode","caller":"v2/main.go:239","message":"Failed to start configuration manager","commit":"c98eb95-dirty","error":"configmap \"pac-config-logging\" not found","stacktrace":"knative.dev/eventing/pkg/adapter/v2.MainWithInformers\n\tknative.dev/eventing@v0.36.7/pkg/adapter/v2/main.go:239\nknative.dev/eventing/pkg/adapter/v2.MainWithEnv\n\tknative.dev/eventing@v0.36.7/pkg/adapter/v2/main.go:158\nknative.dev/eventing/pkg/adapter/v2.MainWithContext\n\tknative.dev/eventing@v0.36.7/pkg/adapter/v2/main.go:133\nmain.main\n\tgithub.com/openshift-pipelines/pipelines-as-code/cmd/pipelines-as-code-controller/main.go:54\nruntime.main\n\truntime/proc.go:250"}
```

Because controller starts first and it doesn't find config map info

Thats the reason create configmap first and then deployment